app/helpers/redirect_destination_helper.rb

```diff
@@ -1,5 +1,5 @@
-module SlugMigrationsHelper
-  def select_options
+module RedirectDestinationHelper
+  def redirect_destination_select_options
```

Comment:
Also.. all helpers are included when rendering views from all controllers these days.. so using a more specific name than `#select_options` is good too :+1: